docs/api/process.md

```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
 When app is started by being passed as parameter to the default app, this
 property is `true` in the main process, otherwise it is `undefined`.
 
+### `process.platform`
+
+Returns the name of the platform the app is running on. `darwin` for macOS, `linux` for Linux and `win32` for Windows.
```

Comment:
The `can be` syntax is also so that the linter can get possible values for the String 👍